package.json

```diff
@@ -97,6 +97,26 @@
           }
         ],
         "priority": "option"
+      },
+      {
+        "viewType": "one.viewer.metadata",
```

Comment:
Could you remove these `Metadata Viewer`-related lines? (1)

Reply:
ok, I`ll apply that part.